src/main/java/ch/njol/skript/conditions/CondIsInvulnerable.java

```diff
@@ -27,18 +28,25 @@
 import ch.njol.skript.doc.Since;
 
 @Name("Is Invulnerable")
-@Description("Checks whether an entity is invulnerable.")
-@Examples("target entity is invulnerable")
-@Since("2.5")
-public class CondIsInvulnerable extends PropertyCondition<Entity> {
+@Description("Checks whether an entity or a gamemode is invulnerable.")
+@Examples({"target entity is invulnerable",
+"loop {_gamemodes::*}:\n"+
+"\tif loop-value is not invulnerable:\n" +
+"\t\tbroadcast \"the gamemode %loop-value% is vulnerable!\""})
+@Since("2.5") // ?
+public class CondIsInvulnerable extends PropertyCondition<Object> {
 	
 	static {
-		register(CondIsInvulnerable.class, PropertyType.BE, "invulnerable", "entities");
+		register(CondIsInvulnerable.class, PropertyType.BE, "invulnerable", "objects");
```

Comment:
```suggestion
		register(CondIsInvulnerable.class, PropertyType.BE, "invulnerable", "entities/gamemodes");
```

Reply:
Actually this needs to check whether the method exists with Skript.methodExists() before adding `gamemodes` to the syntax.